### PR TITLE
Convert FormCommitTemplateSettings to GitExtensionsDialog; fix Dpi issues

### LINKED ISSUE #11343: Broken layout for Commit validation tab in Commit messsage settings window
### Environment

- GitExtensions version: 4.1.0.16698 Build 1fe52a137acef1fbd2b653dea263b44ad5844177
- GIT version: Git 2.42.0.windows.2
- OS version: Microsoft Windows NT 10.0.19045.0
- .NET version: .NET 6.0.24

### Issue description

No elements are visible in default window size:
commit-validation-1

Some elements are visible after stretching the window:
commit-validation-2

### Steps to reproduce

Commit > Commit templates > Edit commit message templates and settings... > Commit validation

### Did this work in previous version of GitExtensions?

Yes, it was fine in 3.5.4.12724

### Diagnostics

_No response_

### PR DESCRIPTION
**Pull request by mdonatas**
Fixes #11343

## Proposed changes

- Convert `FormCommitTemplateSettings` to `GitExtensionsDialog`
- Fix `TabIndex` for controls in the second tab
- Adjust grammar

### Before

**At 100% scale**

image

image

**At 200% scale**

image

image

### After

**At 100% scale**

image

image

**At 200% scale**

image

image

## Test methodology 

- Manually

## Test environment(s) 

- Windows 11
- 100% and 200% scaling

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by mstv — APPROVED** [review]
LGTM, have not run

**Review by gerhardol — APPROVED** [review]
have not run

File: GitUI/CommandsDialogs/CommitDialog/FormCommitTemplateSettings.Designer.cs [hunk 1]
```diff
@@ -253,135 +245,145 @@ private void InitializeComponent()
...
-            checkBoxAutoWrap.AutoSize = true;
-            checkBoxAutoWrap.Location = new Point(319, 55);
-            checkBoxAutoWrap.Name = "checkBoxAutoWrap";
-            checkBoxAutoWrap.Size = new Size(15, 14);
-            checkBoxAutoWrap.TabIndex = 14;
-            checkBoxAutoWrap.UseVisualStyleBackColor = true;
-            // 
             // FormCommitTemplateSettings
             // 
+            AcceptButton = buttonOk;
```
Comment (RussKie) [inline GitUI/CommandsDialogs/CommitDialog/FormCommitTemplateSettings.Designer.cs:374]: 👍 

The "Cancel" button should be set to `CancelButton = buttonCancel;` too.

**Review by RussKie — APPROVED** [review]

**Comment by mdonatas** [comment]
_random :)_

I like how useful Copilot is for non-native English speakers in explaining grammar
image

**Comment by RussKie** [comment]
Thank you

**Pull request merged**